package-lock.json

```diff
@@ -1811,7 +1882,7 @@
         "glob": "7.0.6",
         "htmlparser2": "3.9.2",
         "lodash": "4.17.4",
-        "marked": "0.3.9",
+        "marked": "0.3.6",
```

Comment:
Doh, this unfortunately rolled back out security fix. I'll create a new PR to get this back to `0.3.9`. Will check these hidden `package-lock.json` diffs a little closer in the future.